src/base/ui_object.js

```diff
@@ -188,14 +188,33 @@ export default class UIObject extends BaseObject {
   /**
    * ensures the creation of this ui component
    * @method _ensureElement
+   * @param {Object} options the options object
    * @private
    */
-  _ensureElement() {
+  _ensureElement(options) {
```

Comment:
I am not sure how technically to do that ? the base ui object _ensureElement() contains the logic for html element creation. Do you have any tips ?